docs/CAPABILITY_DATA_SETUP.md

```diff
@@ -68,4 +68,21 @@ Periodic limit per service agreement, for function group "Admin":
 
 - Transactional limit with amount of 1,000,000
 - Privileges "create" and "approve"
-- Functions SEPA CT, US Domestic Wire and US Foreign Wire
\ No newline at end of file
+- Functions SEPA CT, US Domestic Wire and US Foreign Wire
+
+## Notifications setup
+Global notifications are only created be the bank manager using approvals
```

Comment:
created **by**

Reply:
thanks! fixed in commit: https://github.com/Backbase/bb-fuel/pull/30/commits/96933fcec957405afb3b0ad85a9ce08e0dce2272